src/Voice.tsx

```diff
@@ -674,7 +674,7 @@ export class Voice extends EventEmitter {
     );
 
     const selectedDevice =
-      typeof selectedDeviceInfo !== 'undefined'
+      typeof selectedDeviceInfo !== 'undefined' && selectedDeviceInfo !== null
```

Comment:
Now that I think about it, this should never happen. But I also don't think it hurts to have the extra security. WDYT? The native layer shouldn't ever return `selectedDeviceInfo === null`.